projects/personachat/Documentation_License

```diff
@@ -0,0 +1,384 @@
+Attribution 4.0 International
```

Comment:
i think you want this in tasks/personachat for the dataset?
the projects directory is more the set of code to train the task